src/main/rx/fport.c

```diff
@@ -385,7 +385,7 @@ bool fportRxInit(const rxConfig_t *rxConfig, rxRuntimeConfig_t *rxRuntimeConfig)
         telemetryEnabled = initSmartPortTelemetryExternal(smartPortWriteFrameFport);
 #endif
 
-        if (rssiSource == RSSI_SOURCE_NONE) {
+        if (rxConfig->rssi_from_rx_protocol) {
```

Comment:
This should not be dependent on `rssi_from_rx_protocol`, since using FPort implies that the RSSI values sent through the RX protocol should be used, unless the user explicitly configures a different source.

Reply:
As the FPort control frame is the same as SBus according to https://github.com/betaflight/betaflight/files/1491056/F.Port.protocol.betaFlight.V2.1.2017.11.21.pdf I rather assumed the same logic should apply as for SBus. My original argument was that using SBus also implies that the RSSI values sent through the RX protocol should be used, unless the user explicitly configures a different source.

Reply:
Surely we should be consistent between SBus and FPort here? Either they both default to using dropped frames to drive RSSI, or neither should?